index.html

```diff
@@ -2743,7 +2744,7 @@ <h3>Graph Containers</h3>
     </pre>
 
     <p>The example above expresses a <a>named graph</a> that is identified
-      by the <a>blank node identifier</a> <code>_:claim</code>. That
+      by the <a>blank node identifier</a> <code>_:b0</code>. That
```

Comment:
Maybe we should have another example before this one with no blank node identifier. Then, this example can follow and say that the blank node identifier only exists so the graph can be referenced within the same document and that the blank node identifier is meaningless otherwise. I think that would help people understand it.